DEPENDENCIES

```diff
@@ -1328,6 +1328,7 @@ npm/npmjs/@mui/material/5.10.7, MIT, approved, #3244
 npm/npmjs/@mui/private-theming/5.14.20, MIT, approved, #10973
 npm/npmjs/@mui/styled-engine/5.14.20, MIT, approved, #10971
 npm/npmjs/@mui/system/5.14.20, MIT AND CC-BY-3.0, approved, #9905
+npm/npmjs/@mui/types/7.2.12, , restricted, clearlydefined
```

Comment:
Is ticket raised for this restricted dependency?

Reply:
fixed